src/vega.schema.ts

```diff
@@ -68,6 +68,7 @@ export interface VgValueRef {
   mult?: number;
   offset?: number | VgValueRef;
   band?: boolean | number | VgValueRef;
+  test?: string;
```

Comment:
Can we use the type from Vega instead to avoid problems like this in the future?

Reply:
Yes, but it’s beyond the scope of this PR.  Please send a follow up fix.